### PR TITLE
Upgrading to Bootstrap 5.3

### PR DESCRIPTION
**Pull request by planetoftheweb**

### CODE REVIEW

File: docs/columns.html
Comment (jojohumphreys) [inline docs/columns.html:1]: This looks like a good update, but I definitely want to review this further.

File: docs/columns.html [hunk 1]
```diff
@@ -3,13 +3,8 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
-    <script defer src="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"></script>
-    <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css"
-      integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l"
-      crossorigin="anonymous"
-    />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-4bw+/aepP/YC94hEpVNVgiZdgIC5+VKNBQNGCHeKRQN+PtmoHDEXuppvnDJzQIu9" crossorigin="anonymous">
```
Comment (jojohumphreys) [inline docs/columns.html:7]: I'm glad this is all in one line now.

File: docs/index.html [hunk 2]
```diff
@@ -39,7 +34,7 @@
         </div>
       </nav>
 
-      <section class="jumbotron jumbotron-fluid d-flex align-items-center">
+      <section class="bg-body-tertiary py-4 d-flex align-items-center">
         <div class="container text-center">
           <div class="row justify-content-center">
             <div class="col-11 col-sm-10 col-md-8 animated fadeInUp">
```
Comment (jojohumphreys) [inline docs/index.html]: Looks like you're using the animate.css library. You probably want to add the 
Looks like you're using the animate css library, but it's not working.

```suggestion
            <div class="col-11 col-sm-10 col-md-8 animated fadeInUp">
```

**Review by jojohumphreys — APPROVED** [review]
Made some changes, things look pretty good now.

File: docs/index.html (hunk 2 above)
Comment (planetoftheweb) [reply docs/index.html]: Looks like some of your changes took care of this.

File: docs/columns.html
Comment (planetoftheweb) [reply docs/columns.html:1]: Thanks Jojo, you're the best.

**Pull request closed without merging**